cli/clistat/cgroup.go

```diff
@@ -158,28 +169,26 @@ func (s *Statter) cGroupV2CPUTotal() (total float64, err error) {
 }
 
 func (s *Statter) cGroupV1CPUTotal() (float64, error) {
-	periodUs, err := readInt64(s.fs, cgroupV1CFSPeriodUs)
-	if err != nil {
-		// Try alternate path under /sys/fs/cpu
-		var merr error
-		merr = multierror.Append(merr, xerrors.Errorf("get cpu period: %w", err))
-		periodUs, err = readInt64(s.fs, strings.Replace(cgroupV1CFSPeriodUs, "cpu,cpuacct", "cpu", 1))
-		if err != nil {
-			merr = multierror.Append(merr, xerrors.Errorf("get cpu period: %w", err))
-			return 0, merr
-		}
+	var periodUs int64
+	if p, err := readInt64(s.fs, cgroupV1CFSPeriodUs); err == nil {
+		periodUs = p
+	} else if p2, err2 := readInt64(s.fs, strings.Replace(cgroupV1CFSPeriodUs, "cpu,cpuacct", "cpu", 1)); err2 == nil {
+		periodUs = p2
+	} else if errors.Is(err2, fs.ErrNotExist) {
+		return -1, nil
+	} else {
+		return 0, err2
```

Comment:
It's unclear why we prefer err2 over err here. Perhaps a comment would be good?

Also, why do we not differentiate in err between not exist and other errors? Perhaps the issue was permission or something else?

Reply:
Is there a reason why we wouldn't want to use `errors.Join` here? https://pkg.go.dev/errors#Join